applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -640,7 +640,9 @@ static void audio_datapath_i2s_blk_complete(uint32_t frame_start_ts, uint32_t *r
 		size_t size;
 
 		if (ret != prev_ret) {
+#if (CONFIG_STREAM_BIDIRECTIONAL)
 			LOG_WRN("I2S RX overrun. Single msg");
+#endif /* (CONFIG_STREAM_BIDIRECTIONAL) */
```

Comment:
I may have missed something, but what happens if this is the gateway and we are using I2S input? Then we want it to print even though it is not configured for BIDIR.

Reply:
This is actually a good catch :O
It should be e.g. #if ((CONFIG_AUDIO_DEV == GATEWAY) || (CONFIG_STREAM_BIDIRECTIONAL))

Reply:
Added this feature in the revision.